clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonClusterSpec.groovy

```diff
@@ -18,11 +18,12 @@
 package com.netflix.spinnaker.clouddriver.aws.model
 
 import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.clouddriver.jackson.ClouddriverApiModule
 import spock.lang.Specification
 
 class AmazonClusterSpec extends Specification {
   void "should serialize null loadBalancers and serverGroups as empty arrays"() {
-    def objectMapper = new ObjectMapper()
+    def objectMapper = new ObjectMapper().registerModule(new ClouddriverApiModule())
```

Comment:
In `orca` we've created a centralized `OrcaObjectMapper` to [handle registering all the extensions](https://github.com/spinnaker/orca/blob/9d1ff489fd15101c2ddcffe7f654105863086853/orca-core/src/main/java/com/netflix/spinnaker/orca/jackson/OrcaObjectMapper.java#L42) and hide that from consumers
I wonder if we should do something similar for `clouddriver` too, to ensure we don't run into bugs where a particular `ObjectMapper` doesn't have these extensions.

Reply:
There is a centralized one in CloudDriverConfig that I added the new module too. But some unit tests tend to create their own.